admin/services/school-impersonation.service.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+const R = require('ramda')
+
+const schoolDataService = require('../services/data-access/school.data.service')
+const schoolImpersonationEmptyValueValidator = require('../lib/validator/school-impersonation/school-impersonation-empty-value-validator')
```

Comment:
I am not sure what an 'EmptyValue' validator is?  Isn't this the SchooImpersonatorFormValidator?  Why should the implementation be a part of the name?

Reply:
@jon-shipley merged to a single validator